### PR TITLE
test/gtest

### PR DESCRIPTION
**Pull request by richardllara05**
Added basic layout for the project and added support for gtest.

### CODE REVIEW

File: src/Emulator.cpp [hunk 1]
```diff
@@ -15,48 +15,74 @@
...
  *
  */
 
 #include <iostream>
 
-#include <SDL2/SDL.h>
+#include "Emulator.h"
 
-int main(int argc, char* args[])
+Emulator::Emulator()
```
Comment (jaburialeh) [inline src/Emulator.cpp:26]: Is this how you declare an object?

**Review by jaburialeh — APPROVED** [review]
Looks good to me👍

File: src/Emulator.cpp (hunk 1 above)
Comment (richardllara05) [reply src/Emulator.cpp:26]: No, this is actually the constructor. I created the Emulator object in line 7 of Runner.cpp.

Comment (Willygilly) [reply src/Emulator.cpp:26]: It looks like the Emulator.cpp was before the main.cpp. What kind of program Emulator.cpp is suppose to run?

**Review by Willygilly — APPROVED** [review]

**Pull request merged**